package.json

```diff
@@ -45,6 +45,7 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.23.3",
+    "@babel/helper-module-imports": "^7.22.5",
```

Comment:
The three deps moved/copied to the root are to support this: https://github.com/preactjs/signals/blob/e04671469e9272de356109170b2e429db49db2f0/test/browser/babel.js#L10

I'm not sure if that can support being set async w/out race conditions, so I just move the deps up. If they weren't, the tests can still succeed, but there will be a bunch of rather noisy errors printed to the log.

Reply:
Turns out these setup functions are only needed for the React tests. Whoops! Can be filtered out entirely if no react suites are being ran.